exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,14 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+
+var quotientStudents = 15/23 * 100
```

Comment:
You should not hard code the values of the variables, use the variable names instead. e.g. quotientStudents = numberOfStudents / (numberOfStudents + numberOfMentors) * 100

Reply:
Noted, will take care of it.

Reply:
Done, thanks for the feedback